yarn-project/ethereum/src/l1_tx_utils.ts

```diff
@@ -405,14 +442,32 @@ export class L1TxUtils {
     // Ensure priority fee doesn't exceed max fee
     const maxPriorityFeePerGas = priorityFee > maxFeePerGas ? maxFeePerGas : priorityFee;
 
+    if (attempt > 0 && previousGasPrice?.maxFeePerBlobGas) {
+      const bumpPercentage =
+        gasConfig.priorityFeeRetryBumpPercentage! > MIN_BLOB_REPLACEMENT_BUMP_PERCENTAGE
+          ? gasConfig.priorityFeeRetryBumpPercentage!
+          : MIN_BLOB_REPLACEMENT_BUMP_PERCENTAGE;
+
+      // calculate min blob fee based on previous attempt
+      const minBlobFee = (previousGasPrice.maxFeePerBlobGas * (100n + bumpPercentage)) / 100n;
+
+      // use max between current network values and min required values
```

Comment:
Sorry I didn't catch this one earlier, but I believe we're missing the check to not go over `gasConfig.maxGwei` with the blob fee. Given we're duplicating it on every attempt, let's make sure it's in there.